skbio/diversity/alpha/__init__.py

```diff
@@ -146,9 +148,62 @@
 ...                      u'(OTU4:0.75,(OTU5:0.5,((OTU6:0.33,OTU7:0.62):0.5,'
 ...                      u'OTU8:0.5):0.5):0.5):1.25):0.0)root;'))
 >>> otu_ids = ['OTU1', 'OTU2', 'OTU3', 'OTU4', 'OTU5', 'OTU6', 'OTU7', 'OTU8']
->>> faith_pd(counts, otu_ids, tree)
+>>> pd = faith_pd(counts, otu_ids, tree)
+>>> print(round(pd, 2))
 6.95
 
+If you're calculating alpha diversity for more than one sample, you should
+use the ``alpha_diversity`` function, which takes a matrix of per-sample
+count vectors as input. For some metrics (notably ``faith_pd``), this will
+internally use an optimized function so that calling ``alpha_diversity`` on
+all of your samples will be much faster than calling it on each of your samples
+individually.
+
+Create a table containing 7 OTUs and 6 samples:
+
+>>> from skbio.diversity.alpha import alpha_diversity
```

Comment:
Should we have ``alpha_diversity`` and ``beta_diversity`` available from ``skbio.diversity``?